### DISCUSSION TITLE
Module not found: Can't resolve '@/components/Sidebar'

### DISCUSSION THREAD

**Discussion by patrickPauChiHo** [vercel/next.js#65126] [Help]
### Summary

Hi everyone. I am new to Nextjs and I am trying to import my component in my layout but always have the error module not found. I already check my jsconfig and also rebuild the whole app but still. can anyone help me to solve the problem?

**In my app/(dashboard)/layout.js:**
import Sidebar from "@/components/Sidebar";
import { FaBarsStaggered } from "react-icons/fa6";

const layout = ({ children }) => {
  return (
        <>
        <Sidebar />
        {children}
        </>
  );
};

**components/Sidebar:**
const Sidebar = () => {
  return <div className="text-6xl">Sidebar</div>;
};

export default Sidebar;

**jsconfig.json**
{
  "compilerOptions": {
    "paths": {
      "@/*": ["./*"]
    }
  }
}

can anyone solve this problem? please!!
many thanks!!

**Comment by hyjoong** [comment]
If your root directory includes the src folder, try modifying it to `./src/.` If you're not using the src folder, you can try adding an asterisk (*) as follows:
```typescript
"paths" : { 
  "@/*": ["./*"]
}
```
